api-metastore/src/main/java/org/zalando/nakadi/service/EventTypeService.java

```diff
@@ -440,6 +446,12 @@ public void update(final String eventTypeName,
             LOG.error("Failed to wait for timeline switch", e);
             throw new ServiceTemporarilyUnavailableException(
                     "Event type is currently in maintenance, please repeat request", e);
+        } catch (final SchemaValidationException e) {
+            LOG.error("Schema validation failed {}", e.getMessage());
```

Comment:
I would argue this is not an error log. It's not like something broke internally. The user provided invalid data.

Reply:
Reduced to warn